base_external_api_ftp/models/apicli_connection_ftp.py

```diff
@@ -75,7 +77,8 @@ def _download_each_file(self, subdirectory, ftp):
 
     @api.model
     def cron_download_ftp_files(self, subdirectory="/", conn_code=None):
-        conn_code = "DemoFTP"
+        if conn_code is None:
+            conn_code = "DemoFTP"
```

Comment:
@diogocsc  Instead Do not hard code this. Instead add a demo data file, that writes on the default cron record, so that it calls `cron_download_ftp_files(conn_code="DemoFTP")`